test/RadworksGovernor.t.sol

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {ERC20VotesComp} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20VotesComp.sol";
+
+import {IGovernorAlpha} from "src/interfaces/IGovernorAlpha.sol";
+import {RadworksGovernorTest} from "test/helpers/RadworksGovernorTest.sol";
+import {ProposalTest} from "test/helpers/ProposalTest.sol";
+
+abstract contract Constructor is RadworksGovernorTest {
+  function testFuzz_CorrectlySetsAllConstructorArgs(uint256 _blockNumber) public {
+    assertEq(governorBravo.name(), "Radworks Governor Bravo");
+    assertEq(address(governorBravo.token()), RAD_TOKEN);
+
+    assertEq(governorBravo.votingDelay(), INITIAL_VOTING_DELAY);
+    assertLt(governorBravo.votingDelay(), MAX_REASONABLE_TIME_PERIOD);
+
+    assertEq(governorBravo.votingPeriod(), INITIAL_VOTING_PERIOD);
+    assertLt(governorBravo.votingPeriod(), MAX_REASONABLE_TIME_PERIOD);
+
+    assertEq(governorBravo.proposalThreshold(), INITIAL_PROPOSAL_THRESHOLD);
+
+    assertEq(governorBravo.quorum(_blockNumber), QUORUM);
+    assertEq(governorBravo.timelock(), TIMELOCK);
+    assertEq(governorBravo.COUNTING_MODE(), "support=bravo&quorum=bravo");
+  }
+}
+
+abstract contract Propose is ProposalTest {
```

Comment:
This can be handled in a separate pr, but these tests seems to be testing the Alpha Governor rather than the RadworksGovernor. Maybe, reorganizing them could make this distinction clearer.

Reply:
Created an issue for this: https://github.com/ScopeLift/radworks-governor-upgrade/issues/6